classes/03class/exercises/c03-aws01/fabioacp/README.md

```diff
@@ -0,0 +1,153 @@
+# C03-aws01
+
+## Command Execution Output
+- Commands used to create your autoscaling group:
+```bash
+# Get VPC
+❯ aws ec2 describe-vpcs --query "Vpcs[*].VpcId" --output=text
+vpc-8beeebec
+
+# Create a SG
+❯ aws ec2 create-security-group --group-name facp-sg-C03-aws01 --description "SG for exercise C03-aws01" --vpc-id vpc-8beeebec
+{
+    "GroupId": "sg-095154c98505dbfb4"
+}
+# Allow connection from anywhere to port 80
+aws ec2 authorize-security-group-ingress \
+    --group-name facp-sg-C03-aws01 \
+    --protocol tcp \
+    --port 80 \
+    --cidr 0.0.0.0/0
+
+# Create file://facp-user-data.txt
+{
+        echo '#! /bin/bash'
+        echo 'yum update -y'
+        echo 'yum install -y httpd'
+        echo 'curl 169.254.169.254/latest/meta-data/hostname > index.html'
+        echo 'mv index.html /var/www/html/'
+        echo 'systemctl start httpd'
+} >> facp-user-data.txt
+
+# Create a launch configuration 
+❯ aws autoscaling create-launch-configuration \
+  --launch-configuration-name facp-launch-config \
+  --image-id ami-0a58e22c727337c51 \
+  --instance-type t2.nano \
+  --security-groups sg-095154c98505dbfb4 \
+  --user-data file://facp-user-data.txt \
+  --associate-public-ip-address
+
+# Get Subnets
+❯ aws ec2 describe-subnets --query "Subnets[*].[SubnetId, AvailabilityZone]"
+[
+    [
+        "subnet-a50917ec",
+        "ap-southeast-2b"
+    ],
+    [
+        "subnet-f1fc63a9",
+        "ap-southeast-2c"
+    ],
+    [
+        "subnet-d2779ab4",
+        "ap-southeast-2a"
+    ]
+]
+
+  # Create a auto scaling group
+aws autoscaling create-auto-scaling-group \
+--auto-scaling-group-name facp-auto-scaling-group \
+--launch-configuration-name facp-launch-config \
+--min-size 1 \
+--max-size 2 \
+--desired-capacity 2 \
+--vpc-zone-identifier "subnet-d2779ab4,subnet-a50917ec"
+
+```
+
+- Command used to get each asg instance hostname through the webserver
+```bash
+# Get Instance Ids
+❯ aws autoscaling describe-auto-scaling-groups \
+--auto-scaling-group-name facp-auto-scaling-group \
+--query "AutoScalingGroups[*].Instances[*].[InstanceId]"
+[
+    [
+        "i-020306367920ba80c",
+        "i-0e19866a7b6e47962"
+    ]
+]
+
+❯ aws ec2 describe-instances \
+    --instance-ids "i-020306367920ba80c" "i-0e19866a7b6e47962" \
+    --query "Reservations[*].Instances[*].[InstanceId, PrivateDnsName]" \
+    --output=text
+i-020306367920ba80c	ip-172-31-44-35.ap-southeast-2.compute.internal
+i-0e19866a7b6e47962	ip-172-31-6-239.ap-southeast-2.compute.internal
```

Comment:
Here you should verify if the instances' webserver is retrieving the hostname as expected via `curl <public ip>` or via browser.

Reply:
Hey Alan, thanks for reviewing my PR. I just fixed the missing question.

Reply:
Thanks mate, it looks good now ;)